model/clusters_mgmt/v1/manifest_type.model

```diff
@@ -18,4 +18,16 @@ limitations under the License.
 class Manifest {
 	// List of k8s objects to deploy on a hosted cluster.
 	Workloads []Interface
+
+	// Spec of Manifest Work object from open cluster management.
```

Comment:
Useful also to add a link to https://open-cluster-management.io/concepts/manifestwork/

Reply:
Done, thanks!